hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/NSSummaryEndpoint.java

```diff
@@ -48,6 +56,8 @@
 @AdminOnly
 public class NSSummaryEndpoint {
 
+  public static final String COUNT = "count";
```

Comment:
Shouldn't we have createTime here? Also shouldn't that be cTime or creationTime?
As these three belongs together, wouldn't it worth it to create an enum for the values we accept, and validate and parse the parameters based on the enum?